source/_templates/registrations/linux/start_agent.rst

```diff
@@ -0,0 +1,17 @@
+.. Copyright (C) 2020 Wazuh, Inc.
```

Comment:
Those restart agent's methods not represent how to restart Wazuh agents in AIX, Solaris 10/11 and HP-UX.

Reply:
Done